### DISCUSSION TITLE
Fermata not visible in AA

### DISCUSSION THREAD

**Discussion by takinijaki6** [AndreyPavlenko/Fermata#254] [General]
Hi,
maybe someone can tell me what I do wrong. I downloaded and installed *fermata-1.9.1-auto-universal-arm64.apk to my mobile, in AA changed settings giving permission for unatorisher sources, and AA don't see FermataAuto in application menu and in consequence, is not visible after plugging in to screen in car.
Should I install something more? Thanks in advance for any tip

Mobile:
Samsung s20FE
Android 13
AA 9.2.631214
fermata-1.9.1-auto-universal-arm64.apk

**Comment by AndreyPavlenko** [comment]
Use this https://github.com/fcaronte/KingInstaller/releases/tag/1.2 installer to install the app.

**Reply by takinijaki6** [reply]
Hello, 
Thx its work now, but I have problem with voice search in sexrion YT. I confirmed permision in settings, but it doesn't work. Any clue?

**Reply by AndreyPavlenko** [reply]
In the voice control settings, enable the floating button. Then, just long press on the floating button to activate the voice search.
